cmd/util.go

```diff
@@ -0,0 +1,13 @@
+// This file contains common functions that are shared in the lab package
+package cmd
+
+import (
+	"strings"
+)
+
+// textToGitlabText converts text with markdown friendly line breaks
+// See https://gist.github.com/shaunlebron/746476e6e7a4d698b373 for more info.
+func textToGitlabText(text string) string {
```

Comment:
As @zaquestion mentioned before, that's not a specific GitLab behavior, that's basically Markdown's. Wouldn't it be better to name this function something like "textToMdText" or call it LF (line feed as in ASCII table) "toMdLF()" to make it clear that the function only handles \n? I don't know, just thinking out loud here :-D

Reply:
Yup, moved to cmd/util.go and added a test for it too.

Reply:
Pushed new version with changes.